### PR TITLE
Implement Phase 1 maintenance on the repo's root directory

### PR DESCRIPTION
**Pull request by willingc**
Addresses #78 phase 1.

This PR:
- edits README.md by adding commonly used sections, adding badges, and moving usage directions to a new CONTRIBUTING.md file
- adds a CONTRIBUTING.md file seeded with the usage instructions that are currently in the README
- add a CODE_OF_CONDUCT.md file
- adds a minimal CHANGELOG.md file
- adds a LICENSE file and remove src/LICENSE.txt
- remove src/README.md

### CODE REVIEW

**Comment by willingc** [comment]
Will appease codacy style next week.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Code of Conduct
+
+Everyone interacting in the pyOpenSci project's codebases, issue trackers, chat rooms, and communication venues is
```
Comment (lwasser) [inline CODE_OF_CONDUCT.md:3]: I've been meaning to add an [organization wide COC](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) in our [.github repo](https://github.com/pyOpenSci/.github/tree/main) --  I just made an asana task for this as it doesn't belong to a single repo rather to the organization

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+`update-reviews`
+
+This script parses through our (*accepted*) review issues to find packages that have been accepted. It then grabs each
+reviews editor, reviewers and package authors / maintainers. This information allows us to
+
+1. Update a contributor's peer review metadata in the contributors.yml file in the third script.
+2. Update the pyOpenSci website's package listing with the package's DOI, documentation URL.
+3. Update the package's stats including stars, contributors, etc. using the GitHub API
+
+## process\_reviews.py script
```
Comment (lwasser) [inline CONTRIBUTING.md:46]:
```suggestion
## process_reviews.py script
```

File: README.md [hunk 3]
```diff
@@ -1,192 +1,38 @@
-# PyOS Update and Clean Contributor & Peer Review Metadata
+# pyosmeta (was: update-web-metadata)
 
-This repo contains a small module and 3 CLI scripts.
+[![PyPI](https://img.shields.io/pypi/v/pyosmeta.svg)](https://pypi.org/project/pyosmeta/)
```
Comment (lwasser) [inline README.md:3]: @willingc should we rename the repo to be pyosmeta? 

I actually forgot that i pushed this to pypi. I can also setup VCS versioning and github action based release / publish workflow . i've wanted to create that tutorial anyway so minus well do it using hatchling!

**Review by lwasser — COMMENTED** [review]
@willingc this is a great PR. thank you. i'll open up a PR about VCS-based versioning. i thnk it will be really good for me to add that because i do it so infrequently BUT i've had to help other people set it up. i'll create the tutorial while i'm adding that functionality to this repo (as a test to ensure things work!). 

lets :shipit:   :)

**Review by lwasser — APPROVED** [review]

**Comment by lwasser** [comment]
merging this - we can revisit codacy via #80

**Pull request merged**